src/index.html

```diff
@@ -5,8 +5,223 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
+    <header class="menu">
+      <a class="logo" href="">
+        <img
+        src="./images/logo.png"
+        alt="Logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <a href="#" class="nav__link is-active">apple</a>
+        <a href="#" class="nav__link">samsung</a>
+        <a href="#" class="nav__link">смартфоны и телефоны</a>
+        <a
+          href=""
+          class="nav__link"
+          data-qa="nav-hover"
+        >
+        НОУТБУКИ И КОМПЬЮТЕРЫ
+        </a>
+        <a href="#" class="nav__link">гаджеты</a>
+        <a href="#" class="nav__link">планшеты</a>
+        <a href="#" class="nav__link">фото</a>
+        <a href="#" class="nav__link">видео</a>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <div class="card__image"></div>
+          <h1 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+          <p class="card__number">Код товара: 195434</p>
+          <div class="card__rating rating">
+            <div class="rating__stars stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
```

Comment:
```suggestion
            <div class="rating__stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```
Именно цифра 4 в классе определяет сколько активных звездочек должно быть, переиспользуй код из layout_stars